pkg/controller/controller.go

```diff
@@ -81,8 +83,6 @@ func NewController(client kubernetes.Interface,
 			}
 			c.handlePodUpdate(new)
 		},
-
-		DeleteFunc: c.handlePodDelete,
```

Comment:
Why get rid of this?  If some jerk deletes a pod, shouldn't the controller recreate it?

Reply:
Oops!  That was accidental, nice catch!